### PR TITLE
Added blue suns

### LINKED ISSUE #62: blue suns about page
image

### PR DESCRIPTION
**Pull request by duvalincus**
Ran into some issues with the footer, may have to edit the positioning.

image
image

Closes #62.

### CODE REVIEW

File: src/components/Footer.jsx [hunk 1]
```diff
@@ -6,7 +6,7 @@ import logo from "../../public/images/navlogo.webp";
 
 const Footer = () => {
   return (
-    <div className="bg-gradient-to-r from-isa-blue-200 to-isa-blue-100 w-full p-2 relative">
+    <div className="bg-gradient-to-r from-isa-blue-200 to-isa-blue-100 w-full p-2 fixed bottom-0">
```
Comment (seanquiambao) [inline src/components/Footer.jsx]: revert this back to relative. fixing the footer will make the footer stick to window while scrolling, but we don't want that.

File: src/app/about/page.js [hunk 2]
```diff
@@ -2,15 +2,26 @@
...
+    <div className="pb-56 absolute overflow-x-hidden">
       <Image
         src={Logo}
         alt="ISA Icon"
         className="py-8 mx-auto h-[400px] w-[400px]"
       />
+      <Image
+        src={Sun}
+        alt="blue sun"
+        className="absolute bottom-0 -left-[250px] h-[800px] w-[600px]"
```
Comment (seanquiambao) [inline src/app/about/page.js]: - you can use tailwind scale to size the suns without using larger values
- make the suns a little bigger

File: src/app/about/page.js [hunk 3]
```diff
@@ -2,15 +2,26 @@
...
+      />
+      <Image
+        src={Sun}
+        alt="blue sun"
+        className="absolute top-0 -right-[250px] h-[700px] w-[600px]"
```
Comment (seanquiambao) [inline src/app/about/page.js]: see same comments from the other sun

**Review by seanquiambao — CHANGES_REQUESTED** [review]

**Comment by duvalincus** [comment]
image
image

Uses scale-140, and mimicks the hiss layout now.

**Comment by duvalincus** [comment]
image
image

Uses scale-140, mimicks HISS layout.

**Review by seanquiambao — APPROVED** [review]
lgtm!

**Pull request merged**